src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -82,24 +83,35 @@ class ItemRecognition extends Component {
         this.webcam.current.success(() => {
           this.setState({text: 'Snack recognised!', subText: null});
           setTimeout(() => {
-            this.props.history.replace(
-              item.id === 'unknown' ? '/editsnack' : '/confirmitem'
-            );
+            this.props.history.replace('/confirmitem');
           }, 500);
         });
-      } else {
-        if (
-          !this.state.subText &&
-          (Date.now() - this.scanningStartTime) / 1000 >
-            TIMEOUT_IN_SECONDS - SHOW_RETRY_FOR
-        )
-          this.setState({
-            text: "We can't recognise the snack",
-            subText: 'Try turning the snack so the logo is seen by the camera'
-          });
-        if (this.webcam.current)
-          this.webcam.current.requestScreenshot().then(this.handleImg);
+      } else if (
+        (Date.now() - this.scanningStartTime) / 1000 >
+        TIMEOUT_IN_SECONDS
+      ) {
+        // Timed out
+        const suggestions = [];
+        for (let i = 1; i < 4; i++) {
+          suggestions.push(this.props.storeList[items[i].id]);
+        }
```

Comment:
We still need suggestions for if the recognised item is incorrect

Reply:
Lines 78-82 and 94-98 handle the suggestions in either case :)